modules/ROOT/pages/create-endpoint.adoc

```diff
@@ -22,12 +22,12 @@ Ownership of the endpoints (receive or send) is automatically assigned based on
 * *Endpoints: Send to partners*
 * *Endpoints: Target at <host-name>*
 * *Endpoints: Source at <host-name>*
-. <<configure-endpoint,Configure the endpoint>> according the type of endpoint you are creating and the protocol you selected in the *New endpoint: <endpoint-type>* window.
+. <<configure-endpoint,Configure the endpoint>> according to the type of endpoint you are creating and the protocol you selected in the *New endpoint: <endpoint-type>* window.
 . Click *Save*.
 
 == Create and Configure Endpoints using the Message Flow Wizard
 
-When creating endpoint types using the Message Flow wizard, Partner Manager automatically designates the usage. Using the Message Flow Wizard, you can xref:configure-message-flows.adoc[Create and Configure an Inbound Message Flow] or xref:create-outbound-message-flow.adoc[Create and Configure an Outbound Message Flow].
+When creating endpoint types using the Message Flow wizard, Partner Manager automatically designates the usage. Using the Message Flow wizard, you can xref:configure-message-flows.adoc[Create and Configure an Inbound Message Flow] or xref:create-outbound-message-flow.adoc[Create and Configure an Outbound Message Flow].
```

Comment:
This sounds contradictory: PM automatically designates the usage but YOU create and configure?
Do we mean...:

"When you create endpoint types and configure them using the Message Flow wizard, Partner Manager automatically determines whether they are used for inbound or outbound message flows."?

Is that true? I would think that the act of designating a type or configuring it would determine "the usage."
Clarify.

Reply:
I agree - done!